src/components/Wizards/CreateVmWizard/NetworksTab.js

```diff
@@ -239,20 +217,67 @@ export class NetworksTab extends React.Component {
     }
   ];
 
+  getFormFields = pxeNetworks => ({
+    pxeNetwork: {
+      id: 'pxe-nic-dropdown',
+      title: 'PXE NIC',
+      type: 'dropdown',
+      defaultValue: '--- Select PXE NIC ---',
+      choices: pxeNetworks.map(n => n.name),
+      required: true,
+      help: 'Pod network is not PXE bootable'
+    }
+  });
+
+  onFormChange = newValue => {
+    this.setState(state => {
+      state.rows.forEach(row => {
+        row.isBootable = row.name === newValue;
+      });
+      this.publishResults(state.rows);
+      return state.rows;
+    });
+  };
+
   render() {
     const columns = this.getColumns();
     const actionButtons = this.getActionButtons();
 
+    let pxeForm;
+    if (this.props.pxeBoot) {
+      const pxeNetworks = this.state.rows.filter(row => row.network !== POD_NETWORK && row.network !== '');
+      const bootableNetwork = pxeNetworks.find(row => row.isBootable);
```

Comment:
I think it would be better to disable the dropdown if there are no nics to select from

Reply:
@lizsurette what is your opinion on this?

Reply:
IMO, by disabling the dropdown, we would denote that the flow does not allow the value to be selected.
It's different from: there is nothing to be selected from.

I know it's too much for now, but eventually I would better render an action for the user to allow him to change such an "empty" state.